package.json

```diff
@@ -168,6 +168,24 @@
           "description": "Additional headers",
           "default": {},
           "scope": "resource"
+        },
+        "redmine.filter.status": {
+          "type": "array",
+          "description": "Identifiers of status to filter issues",
+          "default": [1,2,3,4,5],
+          "items": {
+            "type": "number"
+          },
```

Comment:
Items type should be like below, so one can filter for status id or `open` literal
```json
"items": {
  "anyOf": [
    {
      "type": "number"
    },
    {
      "type": "string"
    }
  ]
}
```